joplin/static/js/admin.js

```diff
@@ -71,6 +75,53 @@ function configureLivePreview() {
   }, true);
 }
 
+function configurePreviewUpdates() {
+  const editForm = document.querySelector('form#page-edit-form');
+  if (!editForm) {
+    return;
+  }
+
+  editForm.addEventListener('input', _.debounce(updatePreviewWindow, 800));
```

Comment:
This debounce does a pretty good job of giving the user pretty quick feedback. It would be nice as we iterate on this feature to have a spinner or something else to that effect.

Reply:
Great idea cityofaustin/joplin#35